core/templates/components/state-directives/outcome-editor/outcome-feedback-editor.component.html

```diff
@@ -1,18 +1,20 @@
-<style>
-  outcome-feedback-editor .oppia-rule-edit-feedback {
+<div>
```

Comment:
Do you need to wrap this in a div?

Reply:
![css_.png](https://user-images.githubusercontent.com/52493334/109325690-3c6d3580-787c-11eb-9570-1a216a9bcb5d.png)

Reply:
@nithusha I saw wrapping css in div in directive migration docs. Please correct if I have misunderstood.